Dockerfile

```diff
@@ -207,6 +210,9 @@ RUN sed -i -r "/^#?compress/c\compress\ncopytruncate" /etc/logrotate.conf && \
   chown -R clamav:root /var/log/mail/clamav.log && \
   touch /var/log/mail/freshclam.log && \
   chown -R clamav:root /var/log/mail/freshclam.log && \
+  mkdir -p /var/log/supervisor && \
+  touch /var/log/supervisor/supervisord.log && \
+  chown -R syslog:root /var/log/supervisor && \
```

Comment:
Please explain the need for this as well and how it is related to the aliases. It looks like you are trying to solve something, but what?

Reply:
It's not related to aliases, sorry.  I believe I found errors in the logs where supervisord was throwing an error when trying to write to /var/log/supervisor/supervisord.log and it did not exist.  I do apologize for not documenting this one better, I see I didn't even put a comment along with the PR and frankly even I cannot remember exactly where I found this one.

Reply:
I'll amend the first sentence of the previous to say that it *was* tangentially related to aliases in the sense that I found it when trying to discover exactly what was going on with aliases when supervisord brings up the mail service the first time after container creation.  There are some conditions where they don't get added, which appear to be based on how some of the environment variables are set.  I never narrowed it down further than that but thought it important for supervisord to be able to write its logs "out of the box"